recipes/aws-libfabric/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,9 @@
+/* example was taken from https://www.gnu.org/ghm/2011/paris/slides/andreas-enge-mpc.pdf */
+
+#include <rdma/fabric.h>
+#include <iostream>
+
+int main () {
+    const uint32_t version = fi_version();
+    std::cout << "hello libfabric version " << FI_MAJOR(version) << "." << FI_MINOR(version) << "\n";
+}
```

Comment:
Would you mind converting this example to c and rename this file to `test_package.c`.
Then test_package will also test whether the c++ standard library is really not needed.

```suggestion
#include "rdma/fabric.h"

#include <stdio.h>
#include <stdint.h>

int main () {
    const uint32_t version = fi_version();
    printf("libfabric version %d.%d\n", FI_MAJOR(version), FI_MINOR(version));
    return 0;
}
```